package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "wprowadzenie",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
```

Comment:
Na wstępie. Nie znam się na Vue. Za to znam dobrze angulara, angularajs i reacta.

Tutaj widzę, masz komendę serve. Możliwe, że jest tak out of the box w vue, ale przyjęło się, że wersję developerską odpala się komendą start. Więc można ją dodać. Npm ma nawet specjalny alias który pomija skrypt run. npm start -> npm run start.